src/analyses/extractPthread.ml

```diff
@@ -233,22 +233,22 @@ let promela_main : fun_name = "mainfun"
 (* assign tid: promela_main -> 0 *)
 let _ = Tbls.ThreadTidTbl.get promela_main
 
-let fun_ctx ctx f =
-  let ctx_hash =
-    match PthreadDomain.Ctx.to_int ctx with
+let fun_ctx man f =
+  let man_hash =
+    match PthreadDomain.Ctx.to_int man with
     | Some i ->
       i |> i64_to_int |> Tbls.CtxTbl.get |> string_of_int
     | None ->
       "TOP"
   in
-  f.vname ^ "_" ^ ctx_hash
+  f.vname ^ "_" ^ man_hash
```

Comment:
I think here `ctx` was something different (`PthreadDomain.Ctx`), so this (and perhaps other uses of it in this file) shouldn't be renamed.

Curiously, it wasn't renamed in the function name. Did you use something other than plain search and replace?

Reply:
Yes, I tried to keep things that just happened to be named `ctx`, but must have missed a few instances.